source/reference/mongosqld.txt

```diff
@@ -660,17 +656,38 @@ Process Management Options
      - Type
      - Corresponds to
 
-   * - .. setting:: processManagement.service.name
+   * - .. setting:: security.enabled
+     - boolean
+     - :option:`--auth`
+
+   * - .. setting:: security.defaultMechanism
      - string
-     - :option:`--serviceName <mongosqld --serviceName>`
+     - :option:`--defaultAuthMechanism`
 
-   * - .. setting:: processManagement.service.displayName
+   * - .. setting:: security.defaultSource
      - string
-     - :option:`--serviceDisplayName <mongosqld --serviceDisplayName>`
+     - :option:`--defaultAuthSource`
 
-   * - .. setting:: processManagement.service.description
+   * - .. setting:: security.gssapi.hostname
      - string
-     - :option:`--serviceDescription <mongosqld --serviceDescription>`
+     - :option:`--gssapiHostname`
+
+   * - .. setting:: security.gssapi.serviceName
+     - string
+     - :option:`--gssapiServiceName`
+
+.. _msqld-conf-set-parameter-options:
+
+Set Parameter Options
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+   setParameter:
+     polymorphic_type_conversion_mode: <string>
+     type_conversion_mode: <string>
+
+.. include:: /includes/option/option-mongosqld-setParameter.rst
```

Comment:
I think we're getting command line options and configuration file settings confused here.

`--setParameter` is not a command line option.  This ticket is only for the configuration file setting.
You'll need to do the following:

1. Make my changes suggested in "options-mongosqld.yaml" (supplemental link and using `:setting:`)
2. Move everything inside the `description: |` block in "options-mongosqld.yaml" to this line.
3. Undo all changes to the "options-mongosqld.yaml" file.

Reply:
Fixed.